scripts/github-repos/README.md

```diff
@@ -1,53 +1,138 @@
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
+
+'remove_teams'
+Remove all students and child teams in STUDENTTEAM from the org.
+Remove STUDENTTEAM as well.
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+
+'remove_team_repo_access'
+Remove students access to CHIP 10.5 repos that are formed like "PREFIX-FILENAME-[Team number]".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
 ```
 
 This script creates a team that include all stundents in the same semester, eg fa23. 
 Then creates different child teams for them for the chip10.5. At a minimum, 
-you need a CSV file listing all enrolled students with columns 'Email' and 'Team'.  
+you need a CSV file listing all enrolled students with columns 'Username' and 'Team'.  
 The values in the Team columns should be nonnegative integers identifying teams.
 
 Org setting: Assume Base permissions is no permission, each students can only access their team repos.
 
 **Assumption:** You have a GITHUB ORG API key.
 
-### Create a team under the org that includes all students in the same semester
+### Create a team under the org that has all students in the same semester and send invitation
 
 **Use case:** Create a team called STUDENTTEAM, and send invitations 
-to the students in csv file. If STUDENTTEAM exists, delete the team and 
-create a new one. (Team repos still exist) 
-For all students NOT in that team, add/invite into that team.
+to the students in csv file. If STUDENTTEAM exists, delete STUDENTTEAM and 
+create a new one. (Team repos still exist)
+For all students NOT in STUDENTTEAM, add/invite into STUDENTTEAM.
+Team column is not needed for this.
+
+### Create child teams for students for CHIP 10.5
+
+**Use case:** Create child teams for students for CHIP 10.5. 
+
+### Create CHIPS repo for each stedent
+
+**Use case:** Create CHIP repos for each student in STUDENTTEAM.
+The repos' names are formed like  "fa23-[username]-FILENAME". There will 
+be only one collabrator. Add all repos to the gsiteam with admin permission.
 
 ### Create 10.5 repos for each child team 
 
 **Use case:** Each team (as identified by Team column in CSV file)
-gets a repo for chip 10.5.  Repos' names are formed
-by concatenating the prefix, base file name, and the team ID. 
-(eg "fa23-actionmap-04" for team #4) For all students on that team who do 
-NOT already have access to the repo, give them write access on the repo.
+gets a repo for chip 10.5.  Repos' names are formed like "PREFIX-FILENAME-TEAMNUM"

```

Comment:
again, this is confusing because you are using "TEAM" to mean both "a group of people on github in a single github team or child team" and "a small group of students working on a single repo/app".  it might be less confusing to use "GROUP" to mean the second thing since github already uses "TEAM" for the first.